x-pack/solutions/security/plugins/security_solution_serverless/common/pli/pli_features.test.ts

```diff
@@ -4,44 +4,49 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { getProductProductFeatures } from './pli_features';
+import { getEnabledProductFeatures } from './pli_features';
 import * as pliConfig from './pli_config';
 import { ProductLine, ProductTier } from '../product';
 
 describe('getProductProductFeatures', () => {
-  it('should return the essentials PLIs features', () => {
-    // @ts-ignore reassigning readonly value for testing
+  it('returns an empty array if no product types are provided', () => {
+    expect(getEnabledProductFeatures([])).toEqual([]);
+  });
+
+  it('returns features for a single product type', () => {
     pliConfig.PLI_PRODUCT_FEATURES = {
       security: {
+        search_ai_lake: [],
         essentials: ['foo'],
         complete: ['baz'],
       },
     };
 
-    const productFeatureKeys = getProductProductFeatures([
+    const productFeatureKeys = getEnabledProductFeatures([
       { product_line: ProductLine.security, product_tier: ProductTier.essentials },
     ]);
 
     expect(productFeatureKeys).toEqual(['foo']);
   });
 
-  it('should return the complete PLIs features, which includes essentials', () => {
-    // @ts-ignore reassigning readonly value for testing
+  it('returns combined features for multiple product types', () => {
     pliConfig.PLI_PRODUCT_FEATURES = {
       security: {
+        search_ai_lake: [],
         essentials: ['foo'],
         complete: ['baz'],
       },
     };
 
-    const productFeatureKeys = getProductProductFeatures([
+    const productFeatureKeys = getEnabledProductFeatures([
+      { product_line: ProductLine.security, product_tier: ProductTier.essentials },
```

Comment:
I don't think this is a real scenario, `security` product line can be in the `essentials` or `complete` tier,  but not both. We should probably throw an error if we receive this config.

But anyway, I think we can just remove this test case, `complete` tiers no longer extend `essentials`, so we can clean this test too.